### ISSUE TITLE
Geomodel layer

### ISSUE THREAD

**Issue by jgvabo** [equinor/esv-intersection#6]
# Geomodel layer
The component needs to have a geomodel layer. 

## Input
The input should be a stratigraphic column definition (cf. e.g. SMDA https://smda.equinor.com, this contains a stratigraphic hierarchy and color defs) and a set of named curves representing horizons in the geomodel. Erosional surfaces not in the stratigraphic column should be given as named curves with an associated drawing color

**Comment by andreabrambilla** [comment]
We need more details about this
We need to define the input data format for this layer
This probably need to be split into smaller issues

**Comment by stalenestas** [comment]
@farzadfz @thuve @andreabrambilla @Brynjulf 
The issue I was trying to bring up with the server side rendering of the geomodel, was a issue with server side rendering of WebGL. We have now used PIXIJS to render the shapes. The PIXIJS was supporting both WebGL and Canvas rendering methods, the WebGL is more difficult to handle in headless mode / server side. 
Canvas is easier, but does not support all the cool stuff that WebGL can do. The newest PIXI has even gone so far as to extract the previously used fallback mode of Canvas-support, into a repository called pixijs-legacy.

Bottom line, if we do need to support server side rendering. We do need to develop for both, and making sure they look the same in Canvas and WebGL.
